h/static/styles/group-form.scss

```diff
@@ -73,6 +74,14 @@
     }
   }
 
+  .group-form-document-list {
+    text-align:center;
+  }
+
+  .group-form-document-list li {
+    margin-top: .5em;
```

Comment:
I'm generally preferring `px` units in new styling as this makes it easier to ensure fidelity with mockups.